Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
-name = "term_grid"
+name = "uutils_term_grid"
 description = "Library for formatting strings into a grid layout"
 
 authors = ["uutils developers", "Benjamin Sago <ogham@bsago.me>"]
 documentation = "https://docs.rs/uutils_term_grid/"
 license = "MIT"
 readme = "README.md"
 repository = "https://github.com/uutils/uutils-term-grid"
-version = "0.2.0"
+version = "0.3.0"
 edition = "2021"
 rust-version = "1.56"
```

Comment:
For an initial release it might make sense to use a more current Rust version.

Reply:
Yes makes sense, I'll match the coreutils MSRV (and do a quick check on the eza MSRV).